src/handlebars/supported_platforms.handlebars

```diff
@@ -45,6 +45,7 @@
                     <th>x64</th>
                     <th>ppc64le</th>
                     <th>s390x</th>
+                    <th>aarch32</th>
```

Comment:
Does the API serve up as aarch32 or as arm32?  I have seen both ways of naming this platform... we should likely attempt to consolidate on 1 way of naming...

Reply:
Neither ... The API uses `arm` at the moment for 32-bit ARM but I agree this was the one thing I was undecided about, but chose to make it the same as the other columns that were in the file for now.